__tests__/libs/ontuml2gufo/helpers.ts

```diff
@@ -1,5 +1,5 @@
 import { ModelManager } from '@libs/model';
```

Comment:
This file should not exist anymore, just `__tests__/lib/ontouml2gufo/helpers.ts`. 
It is the old and incorrect library name.